### DISCUSSION TITLE
Welcome to SnapGrid Discussions!

### DISCUSSION THREAD

**Discussion by Vijaykv5** [Vijaykv5/Image-Searcher#69] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Vijaykv5** [comment]
Changed the Name of the App to 
### SnapGrid

**Comment by juninhokaponne** [comment]
Hello, I have a question regarding the choice of using Tailwind CSS in the project, why was this tool chosen in the beginning?

**Comment by SuyashRajput** [comment]
@Vijaykv5 I have worked with infinite scroll and lazy loading stuff before
Was thinking to implement infinite scroll and load the images only when they come to view using intersection observer to improve the performance
if you think this would be a good contribution I will go ahead with the work

**Comment by JavidSumra** [comment]
Hey @Vijaykv5 for one Feature Implementation I want to use Material UI or Headless UI Can i use it?

**Comment by Vijaykv5** [comment]
You can use headless UI

**Reply by JavidSumra** [reply]
ok @Vijaykv5  please reply on my issue no #198 and #196

**Comment by JavidSumra** [comment]
Hello @Vijaykv5, I'd like to discuss the backend for the project. I'm considering using Node.js for it. Could you please provide details on the folder structure so I can get started?

**Reply by JavidSumra** [reply]
Hey @Vijaykv5 please reply to this discussion

**Reply by Vijaykv5** [reply]
for what feature are you thinking of backend?

**Reply by JavidSumra** [reply]
Signup and sign in and @Vijaykv5 please check out my PR for PWA.

**Reply by muhammedsirajudeen** [reply]
@Vijaykv5 @JavidSumra i guess we could leverage serverless functions from netlify instead of additional layers of infrastructure

**Reply by JavidSumra** [reply]
Hey @Vijaykv5, we can utilize `Clerk` for implementing the login/signup feature.

**Reply by Vijaykv5** [reply]
Its actually a good way to start with. 
but not good for professionally large scale projects.

Let's now go with clerk. @JavidSumra Would you like to try it?

**Reply by JavidSumra** [reply]
Yeah @Vijaykv5
